src/System.Text.Http.Parser/BadHttpRequestException.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+
+namespace System.Text.Http.Parser
+{
+    internal static class StatusCodes
+    {
```

Comment:
@davidfowl, do you know why you are not using the System.Net enum? I assume because it did not exist when you guys wrote the code?

@stephentoub, are you going to add these values?